doc/tutorials/modeling-threat-intel-ctim.md

```diff
@@ -0,0 +1,702 @@
+- **producer**: The name of the party or organization that produced the indicator entity, distinct from the `source` of the threat intelligence.
+
+##### 1.3.1.2: Optional Indicator Fields
+
+These are the most important of the many optional fields.  A full list of optional fields can be found in the [Indicator Schema](https://github.com/threatgrid/ctim/blob/master/src/ctim/schemas/indicator.cljc).
+- **confidence**: Must be one of `["Info", "Low", "Medium", "High", "None", "Unknown"]`.
+- **severity**: Must be one of `["Info", "Low", "Medium", "High", "None", "Unknown"]`.
+
+
+#### 1.3.2: Types of Indicators
+
+Broadly speaking, indicators come in two types:
+
+1. **Pattern or rule based** indicators, such as those you would execute in an expert system (such as Threat Grid), or inside of a rule engine (such as Snort), or even a next generation firewall.
+2. **Observable based feed and watchlist** indicators, such as a feed containing known malicious IP addresses, or a feed containing URLs that are used in botnet Command and Control networks, or perhaps a feed containing known malicious SSL certificate hashes.  These tend to contain lists of observables, and are updated periodically.  How much stock you place in these feeds can depend on the age of their contents, the reputation of their source, and their false positive rate.  Unlike the pattern or rule based indicators, observable-based feeds and watchlists often obscure the precise combinations of rules or patterns that led to the inclusion of a given observable in the feed.
+
+#### 1.3.3: Example Indicator
+
+```json
+{
+  "type": "indicator",
+  "source": "Modeling Threat Intelligence in CTIM Tutorial",
+  "source_uri": "https://github.com/threatgrid/ctim/blob/master/src/doc/tutorials/modeling-threat-intel-ctim.md",
+  "title": "Example Indicator Title",
+  "short_description": "Example indicator entity, provided for purposes of illustrating the correct construction of indicators in a CTIM tutorial.",
+  "valid_time": {
+      "start_time": "2019-02-28T00:00:00.000Z",
+      "end_time": "2525-01-01T00:00:00.000Z"
+  },
+  "confidence": "None",
+  "severity": "None",
+  "tags": ["example"],
+  "tlp": "white",
+  "producer": "Cisco Systems",
+  "external_ids": [
+      "ctim-tutorial-indicator-5206f31d14f7b1965dc97c1ec8febfbe45439e8872ff19782f6ac7c49a0ffc68"
+  ],
+  "id": "transient:ctim-tutorial-indicator-5206f31d14f7b1965dc97c1ec8febfbe45439e8872ff19782f6ac7c49a0ffc68"
+}
+```
+
+----
+
+### 1.4: Judgement Entities
+
+A judgement about the intent or nature of an observable.  For example, is it malicious, meaning is is malware and subverts system operations?  It could also be clean and be from a known benign, or trusted source.  It could also be common, something so widespread that it's not likely to be malicious.
```

Comment:
```suggestion
A judgement about the intent or nature of an observable.  For example, is it malicious, meaning it is malware and subverts system operations?  It could also be clean and be from a known benign, or trusted source.  It could also be common, something so widespread that it's not likely to be malicious.
```

Reply:
I think this is copy pasta from the Judgement schema, so we might want to correct it there as well.  Again, eagle eye!